stubs/CoreGenericFunctions.phpstub

```diff
@@ -136,14 +136,54 @@ function array_flip(array $array)
  *
  * @param TArray $array
  *
- * @return (TArray is array<empty, empty> ? null : TKey|null)
+ * @return (TArray is array<empty,empty> ? null : (TArray is non-empty-list ? int<0,max> : (TArray is non-empty-array ? TKey : mixed)))
```

Comment:
Mixed should be replaced by TKey|null

You excluded the empty array and the non-empty-array but it doesn't mean there's nothing left. You still have the simple array type where you don't know if there is content in the array. And in this case you still can't say if it will return null or TKey